src/dnsvizor_web_interface.md

```diff
@@ -0,0 +1,99 @@
+# Web Interface & API
+
+DNSvizor exposes a web-based management interface and a DNS-over-HTTPS (DoH) resolver.
+
+## Authentication
+Access to sensitive endpoints (like the blocklist or configuration) requires authentication.
+
+- **Password Setup:** You must pass the `--password` argument at startup. If omitted, endpoints requiring authentication will not be accessible.
```

Comment:
"You must.. If omitted.." sounds to me that I don't "must", but I can omit that.